src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import Home from "./pages/Home";
+import About from "./pages/About";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import Vans from "./pages/vans/Vans";
+import VanDetail from "./pages/vans/VanDetail";
+import "./server";
+import Layout from "./components/Layout";
+import Dashboard from "./components/host/Dashboard";
+import Income from "./components/host/Income";
+import Reviews from "./components/host/Reviews";
+import HostLayout from "./components/HostLayout";
+import HostVans from "./components/host/HostVans";
+import HostVanDetails from "./components/host/HostVanDetails";
+import HostVanInfo from "./components/host/HostVanInfo";
+import HostVanPricing from "./components/host/HostVanPricing";
+import HostVanPhotos from "./components/host/HostVanPhotos";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Home />}></Route>
```

Comment:
Since there are no nested routes inside this Route component, you can make it self-closing,
```suggestion
          <Route index element={<Home />}/>
```